pglast/printers/ddl.py

```diff
@@ -96,9 +96,14 @@ def alter_enum_stmt(node, output):
     output.write("ALTER TYPE ")
     output.print_name(node.typeName)
     if node.newVal:
-        output.write("ADD VALUE ")
-        if node.skipIfNewValExists:
-            output.write("IF NOT EXISTS ")
+        if node.oldVal:
+            output.write("RENAME VALUE ")
+            output._write_quoted_string(node.oldVal.value)
```

Comment:
Is there a particular reason `output.write_node(node.oldVal)` does not work here? In general, printers should rarely need to call _internal_ stream's `_underscore_prefixed_methods()` directly...

Reply:
I wanted to do the same as what was done a few lines below. Git blaming through it revealed I was the culprit there too last year :)

Because oldVal and newVal are not wrapped in String nodes, they are not quoted by print_node and as such I need to call a quoting function.
Maybe something else should be called ? Or the write_quoted_string method should be made public for those use cases ?

Reply:
Oh, I see, they are not "nodes"... so yes, we should probably promote that method.
Will try to do that when I'll merge this.